Vostok.Metrics.AspNetCore/Vostok.Metrics.AspNetCore.csproj

```diff
@@ -1,28 +1,40 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
-  
   <Import Project="..\..\vostok.devtools\library-common-props\Main-Project.props" />
   <Import Project="..\..\vostok.devtools\git-commit-to-assembly-title\Vostok.Tools.GitCommit2AssemblyTitle.props" />
-
   <PropertyGroup>
     <Configurations>Debug;Release</Configurations>
   </PropertyGroup>
-
   <PropertyGroup>
     <VersionPrefix>0.0.1</VersionPrefix>
   </PropertyGroup>
-
   <PropertyGroup>
     <Title>Vostok.Metrics.AspNetCore</Title>
     <Description>Metrics collection for AspNetCore applications.</Description>
   </PropertyGroup>
-
   <PropertyGroup>
     <PackageProjectUrl>https://github.com/vostok/metrics.aspnetcore</PackageProjectUrl>
     <PackageLicenseUrl>https://github.com/vostok/metrics.aspnetcore/blob/master/LICENSE</PackageLicenseUrl>
     <PackageTags>vostok vostok.metrics</PackageTags>
     <PackageReleaseNotes>https://github.com/vostok/metrics.aspnetcore/blob/master/CHANGELOG.md</PackageReleaseNotes>
     <RepositoryUrl>https://github.com/vostok/metrics.aspnetcore</RepositoryUrl>
   </PropertyGroup>
-  
+  <ItemGroup>
+    <Compile Include="..\..\vostok.commons.collections\Vostok.Commons.Collections\DictionaryExtensions.cs" Link="Commons\DictionaryExtensions.cs" />
+    <Compile Include="..\..\vostok.commons.environment\Vostok.Commons.Environment\RuntimeDetector.cs" Link="Commons\RuntimeDetector.cs" />
+    <Compile Include="..\..\vostok.commons.helpers\Vostok.Commons.Helpers\Observable\BroadcastObservable.cs" Link="Commons\BroadcastObservable.cs" />
+    <Compile Include="..\..\vostok.metrics.system\Vostok.Metrics.System\Helpers\ConcurrentCounter.cs" Link="Commons\ConcurrentCounter.cs" />
+    <Compile Include="..\..\vostok.metrics.system\Vostok.Metrics.System\Helpers\DeltaCollector.cs" Link="Commons\DeltaCollector.cs" />
```

Comment:
That's quite weird that the `vostok.metrics.aspnetcore` depends on the `vostok.metrics.system` package. Luckily, that's not the case for Nuget consumers, so that's probably fine. 

But I still think that perhaps we could move this class to one of the `commons` libraries, so it is more consistent with other `compile include` usages in Vostok, and we can also get rid of unnecessary dependency that way.

What about `Vostok.Commons.Helpers`?

Reply:
Move to Vostok.Commons.Helpers:
- `DeltaCollector`
- `ConcurrentCounter`
- `EventHelper`

Reply:
https://github.com/vostok/commons.helpers/pull/10